perf/benchmark_setup_linux.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
```

Comment:
Some extra error handling will be required for commands that can't execute - might be worth running shellcehck over this for guidance,